docs/vmanomaly.md

```diff
@@ -135,3 +135,59 @@ It is also possible to split up config into multiple files, just list them all i
 ```sh
 python3 -m vmanomaly model_prophet.yaml io_csv.yaml scheduler_oneoff.yaml
 ```
+
+### Licensing
+
+Starting from v1.5.0 vmanomaly requires a license key to run. You can obtain a trial license
+key [here](https://victoriametrics.com/products/enterprise/trial/).
+
+The license key can be passed via the following command-line flags:
+```
+  --license LICENSE     See https://victoriametrics.com/products/enterprise/
+                        for trial license
+  --license-file LICENSE_FILE
+                        See https://victoriametrics.com/products/enterprise/
+                        for trial license
+  --license-verify-offline LICENSE_VERIFY_OFFLINE
+                        Force offline verification of license code
+```
+
+Usage example:
+```
+python3 -m vmanomaly --license-file /path/to/license_file.yaml config.yaml
+```
+
+In order to make it easier to monitor the license expiration date, the following metrics are exposed:
+```
+# HELP vm_license_expires_at When the license expires as a Unix timestamp in seconds
+# TYPE vm_license_expires_at gauge
+vm_license_expires_at 1.6963776e+09
+# HELP vm_license_expires_in_seconds Amount of seconds until the license expires
+# TYPE vm_license_expires_in_seconds gauge
+vm_license_expires_in_seconds 4.886608e+06
+```
+
+You can find example alerts for [vmalert](https://docs.victoriametrics.com/vmalert.html):
+```yaml
```

Comment:
please link to monitoring documentation for vmanomaly and describe more the need of this aletrs

Reply:
Added monitoring section and linked it from this part.

Reply:
we have monitoring docs here as well
https://github.com/VictoriaMetrics/vmanomaly/blob/master/docs/monitoring/monitoring.md

Reply:
i ment monitoring section for vmanomaly as far it will expose this metrics and link to how to setup alerting (link to vmalert)